packages/openapi-to-graphql/src/index.ts

```diff
@@ -491,11 +586,45 @@ function getFieldForOperation(
      */
     requestPayloadDef: operation.payloadDefinition,
     parameters: operation.parameters,
-
     operation,
     data
   })
 
+  if (operation.isSubscription) {
+    const responseSchemaName = operation.responseDefinition
```

Comment:
If operationType is Subscription, then create a specific field with subscribe and optionally resolve